code/modules/clothing/clothing.dm

```diff
@@ -41,6 +41,11 @@
 	/// How many zones (body parts, not precise) we have disabled so far, for naming purposes
 	var/zones_disabled
 
+	/// If on the hands, the multiplier we will apply to our wearer's grab resilience mult, which decreases the chance of victims escaping grabs.
+	var/grab_resilience_mult = 1
+	/// If on the hands, the increment we will apply to our wearer's grab resilience mult, which decreases the chance of victims escaping grabs.
+	var/grab_resilience_increment = 0
```

Comment:
Is this var actually used anywhere? The increment is meant to be a flat addition to resilience multi, correct? 

I'm not sure if that comment explains that super well, it would be easy to assume the var above it is basically the same thing even though they are applied completely differently. Particularly since there is no guidelines for what values would be acceptable. Is it even a necessary var in the first place?

Reply:
yeah thats unused

frankly this code is kinda crusty, ill get to touching it up in a bit

Reply:
i forgot to touch it up crap